permissions/plugin-gitee.yml

```diff
@@ -0,0 +1,7 @@
+---
+name: "gitee-plugin"
```

Comment:
Pretty sure this is wrong as we don't admit new plugins not following the naming convention.